server/services/v1/auth/auth0.go

```diff
@@ -289,6 +289,29 @@ func (a *auth0) ListAppKeys(ctx context.Context, req *api.ListAppKeysRequest) (*
 	}, nil
 }
 
+func (a *auth0) DeleteAppKeys(ctx context.Context, project string) error {
```

Comment:
so the caller will call this? Or should we just call this when we receive the DeleteProject request?

Reply:
Nvm, ignore my comment. I missed the above call.

Reply:
caller will call delete-project - the handler internally calls this function to also delete the app-keys associated with that project. this is not a public facing handler.